maro/simulator/scenarios/oncall_routing/business_engine.py

```diff
@@ -558,3 +580,7 @@ def post_step(self, tick: int) -> bool:
                         f"remaining plan: {len(route.remaining_plan)}"
                     )
         return is_done
+
+    @property
+    def estimated_duration_predictor(self) -> EstimatedDurationPredictor:
```

Comment:
No need for this property if we return self._xxx_predictor directly in Decision Event Payload